flow/Makefile

```diff
@@ -672,7 +672,7 @@ finish: $(LOG_DIR)/6_report.log \
         $(RESULTS_DIR)/6_final.v \
         $(RESULTS_DIR)/6_final.sdc \
         $(GDS_FINAL_FILE)
-	-@$(UTILS_DIR)/genElapsedTime.py -d "$(LOG_DIR)"
+	$(UNSET_AND_MAKE) elapsed
```

Comment:
This is an example of how `$(MAKE)` would cause problems, but where `$(UNSET_AND_MAKE)` works as expected.